src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry.Test/ScenarioTests/ContainerRegistryTests.ps1

```diff
@@ -0,0 +1,167 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test New-AzureRmContainerRegistry, Get-AzureRmContainerRegistry, Update-AzureRmContainerRegistry, and Remove-AzureRmContainerRegistry.
+#>
+function Test-AzureContainerRegistry
+{
+    # Setup
+    $resourceGroupName = Get-RandomResourceGroupName
+    $registryName = Get-RandomRegistryName
+    $location = "southcentralus"
```

Comment:
@djyou any reason you don't get the location from the `Common.ps1` script? We want to avoid hard-coding location values in tests as it makes it difficult for other users to re-record them in the future